dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -37,8 +39,21 @@
     public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGitHubUtil)
             throws IOException, InterruptedException {
         loadDockerfileGithubUtil(dockerfileGitHubUtil);
-        Set<Map.Entry<String, JsonElement>> imageToTagStore =
-                this.dockerfileGitHubUtil.getGitHubJsonStore(ns.get(Constants.STORE)).parseStoreToImagesMap(dockerfileGitHubUtil, ns.get(Constants.STORE));
+        String store = ns.get(Constants.STORE);
+        ImageStoreUtil imageStoreUtil = getImageStoreUtil();
+        try {
+            ImageTagStore imageTagStore = imageStoreUtil.initializeImageTagStore(this.dockerfileGitHubUtil, store);
+            List<ImageTagStoreContent> imageNamesWithTag = imageTagStore.getStoreContent(dockerfileGitHubUtil, store);
+            Integer numberOfImagesToProcess = imageNamesWithTag.size();
+            List<ProcessingErrors> imagesThatCouldNotBeProcessed = processImagesWithTag(ns, imageNamesWithTag);
+            printSummary(imagesThatCouldNotBeProcessed, numberOfImagesToProcess);
+        } catch (URISyntaxException e) {
+            log.error("Could not initialize the Image tage store. Exception: ", e.getMessage());
+        }
```

Comment:
This is good.  I but I didn't intend for the checked exception to be caught and swallowed here.  I should have been more clear.  We should catch the checked exception and yest we can log the exception.  But we should wrap it in an unchecked exception and throw that.  Then we aren't handling exceptions prematurely but we also aren't throwing checked exceptions that break the method signature.  Ideally we create `DFIU` specific unchecked exceptions and throw those so we have abstracted our dependencies from our customers.